digitalocean/blog.yaml

```diff
@@ -0,0 +1,46 @@
+# https://www.digitalocean.com/docs/app-platform/references/app-specification-reference/
+domains:
+- domain: blog.whatwg.org
+  type: PRIMARY
+  zone: whatwg.org
+name: blog
+region: nyc
+services:
+- dockerfile_path: Dockerfile
+  envs:
+  - key: WORDPRESS_DB_HOST
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.HOSTNAME}:${db-mysql-nyc3-38716.PORT}
+  - key: WORDPRESS_DB_NAME
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.DATABASE}
+  - key: WORDPRESS_DB_USER
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.USERNAME}
+  - key: WORDPRESS_DB_PASSWORD
+    scope: RUN_TIME
+    value: ${db-mysql-nyc3-38716.PASSWORD}
+  - key: WORDPRESS_EXTRA_CONFIG
+    scope: RUN_TIME
+    value: |
+      define('DB_CHARSET', 'utf8mb4');
```

Comment:
Is it utf8mb4 in the current blog hosting, or are you fixing this in the migration? I recall fixing encodings in the previous migration, but don't know if the result was utf8mb4.

Reply:
It's the current blog hosting.